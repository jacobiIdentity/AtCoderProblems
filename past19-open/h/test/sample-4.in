6 302934
614 490 585 613 420 945
