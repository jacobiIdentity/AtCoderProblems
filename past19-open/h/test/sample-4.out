Yes
420+490x613+585+614+945
